tailormap-components/projects/core/src/lib/shared/formconfig-repository/formconfig-repository.service.ts

```diff
@@ -32,7 +35,12 @@ export class FormconfigRepositoryService {
     private domainRepo: DomainRepositoryService,
     private tailorMap: TailorMapService,
   ) {
-    this.http.get<FormConfigurations>(this.tailorMap.getContextPath() + '/action/form')
+    let httpParams: HttpParams = new HttpParams();
```

Comment:
Small tip: if you create a new object which you are not going to change its not really needed to also specify the type.

Also, these HttpParams are chainable, for example, this is the same:

`const httpParams = new HttpParams().set('application', '' + this.tailorMap.getViewerController().app.id);`

Reply:
ja... wat Geert zegt :wink: